src/mcse.jl

```diff
@@ -1,72 +1,127 @@
+Base.@irrational normcdf1 0.8413447460685429486 StatsFuns.normcdf(big(1))
+Base.@irrational normcdfn1 0.1586552539314570514 StatsFuns.normcdf(big(-1))
```

Comment:
Defining `@irrational`s outside of base is actually type piracy - it will add implementations for `Irrational{:normcdf1}` etc. which is not owned by us. We decided to do it anyways in IrrationalConstants but core developers told us `@irrational`/`Irrational` was not supposed to be used outside of base. Hence if possible I'd like to avoid these definitions here:
```suggestion
```

Reply:
Ah, okay. In this case we round to an `Int` eventually anyways, so it's fine to use a `Float64`.
```suggestion
const normcdf1 = 0.841344746068543  # StatsFuns.normcdf(1)
const normcdfn1 = 0.15865525393145702  # StatsFuns.normcdf(-1)
```